lib/jobs.rb

```diff
@@ -27,8 +27,15 @@ def self.establish_connection!
     end
   end
 
-  def self.search(options)
-    @usajobs_api_connection.get(@endpoint, options).body if query_eligible?(options[:Keyword])
+  def self.job_scrub(keyword)
```

Comment:
The naming of this method is a little confusing to me. Its purpose is to scrub keywords from the original query, so maybe we could call it something like `scrub_keywords(query)`